icd/api/include/compiler_solution.h

```diff
@@ -111,6 +114,9 @@ struct ComputePipelineCreateInfo
     bool                                   freeWithCompiler;
     Util::MetroHash::Hash                  basePipelineHash;
     PipelineCreationFeedback               pipelineFeedback;
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 46
+    PipelineCreationFeedback stageFeedback[1];
```

Comment:
Why not make it a normal variable instead of a 1-element array?

Reply:
updated